src/content/docs/zh-cn/tutorial/0-introduction/1.mdx

```diff
@@ -0,0 +1,58 @@
+---
+type: tutorial
+title: 关于本教程
+description: 如何开始学习 "建立你的第一个 Astro 博客 "的教程。
+---
+import Badge from '~/components/Badge.astro';
+import Checklist from '~/components/Checklist.astro';
+import Box from '~/components/tutorial/Box.astro';
+
+
+## 我需要知道什么前置知识？
+
+如果你对 **HTML**、**Markdown**、**CSS** 和一点 **JavaScript** 有一些基本的了解，那么你就可以开始了！ 只需按照说明进行操作，你就可以完成整个教程。 Astro 适合所有人！ 🧑‍🚀 👩‍🚀 👨‍🚀
+
+你还需要一个 [GitHub](https://github.com)（或类似的）账号来将你的项目部署到网络上。
+
+<details>
+<summary>如何使用每页底部的清单？</summary>
+
+你把它们勾掉!
+
+在每个页面的末尾，你会发现一个可点击的任务清单，你现在应该能够做到。勾选这些项目，就可以在 "教程追踪 "中看到你的进展。
+
+使用跟踪器是可选的，但如果你在多次访问中完成教程，它可以帮助你记住你的位置。你也可以把一些核对表留空，以提醒你以后要重温的单元。
+
+(这些数据只保存在你的浏览器的本地存储中，在其他地方无法使用。没有数据会被发送到 Astro，也不会被 Astro 存储）。
+</details>
+
+<details>
+<summary>我已经知道第一单元该做什么了。我可以跳过吗？</summary>
+
+你可以使用[Unit 1](/en/tutorial/1-setup/)里面的课程，确保你拥有完成教程所需的开发工具和账户。它将指导你创建一个新的 Astro 项目，将其存储在 GitHub 上，并将其部署到 Netlify。
+
+如果你[创建一个新的、空的Astro项目](/en/install/auto/)并对你的设置感到满意，你可以安全地跳到[第二单元](/en/tutorial/2-pages/)，你将开始在你的项目中制作新页面。
```

Comment:
Thanks hard work! @KurumiFrost .
There are some minor issues that need to be changed here.
```suggestion
如果你[已经创建了一个新的、空的Astro项目](/zh-cn/install/auto/)并对你的设置感到满意，你可以安全地跳到[第二单元](/en/tutorial/2-pages/)，你将开始在你的项目中制作新页面。
```